### ISSUE TITLE
Default gallery style missing

### ISSUE THREAD

**Issue by mekanix** [FineUploader/react-fine-uploader#50]
The <Gallery /> in documentation looks totally different. Could it be something about my webpack config?

![galery](https://cloud.githubusercontent.com/assets/610855/19851873/5abcf704-9f60-11e6-8c08-1c53a42ad001.png)

**Comment by rnicholus** [comment]
Yes, you need the style loader for webpack since CSS files are imported by a few components, namely gallery. I should probably update the docs to mention this.

**Comment by rnicholus** [comment]
...or I should update my code to not explicitly import CSS files and instead require integrators to do this.

**Comment by mekanix** [comment]
Currently, I have this as loader:
```
const localIdentName = '[path]___[name]__[local]___[hash:base64:5]';
```
and
```
      {
        test: /\.css$/,
        loaders: [
          'style?sourceMap',
          `css?sourceMap&modules&importLoaders=1&localIdentName=${localIdentName}`,
          'postcss',
        ],
      },
```
What would be the proper way? Thank you!

**Comment by rnicholus** [comment]
You can use the webpack config in this project as an example

**Comment by mekanix** [comment]
Just for reference, if anyone else has the same problem. Omit `localIdentName` in css loader config and everything works just fine. Thank you for the help!

**Comment by rnicholus** [comment]
I'm going to re-open this as a reminder to myself to deal with this. I'm leaning towards removing the explicit CSS imports in the library and requiring integrators to import as they see fit. I'd hate to have Webpack-specific requirements listed in the docs.

**Comment by rnicholus** [comment]
This is fixed in [v0.4.0](https://github.com/FineUploader/react-fine-uploader/releases/tag/0.4.0).
